### ISSUE TITLE
put better documentation for novice zealots.. 

### ISSUE THREAD

**Issue by mckaygerhard** [antirek/owo-phone.js#3]
i arrives here due i found a comment said "too large and complicated" in a google group and 
seems your app demo are more understandable.. 

but it's lacks of a good tutorial.. currently i spend around 13 hours configuring and setup the environment.. now i'm at the point athat have changed the ws uri but not got to work.. 

of course we need a good knowledge of webrtc but i think we can setup/buil/create a step by step guide using your example, debuggin at the F12 in firefox show too many errors : 

```
SIP/2.0 403 Forbidden
Via: SIP/2.0/WSS lr3e67ai39jv.invalid;rport=21970;received=192.168.10.3;branch=z9hG4bK3878731
To: <sip:1060@192.168.10.2>;tag=9d079adf30be9ee6222f98d845a8175e.b337
From: <sip:1060@192.168.10.2>;tag=ukdv982g5n
```

asterisk are working correctly:

```
asterisk -rvvv -x 'http show status'
HTTP Server Status:
Prefix: 
Server: Asterisk/13.14.1
Server Enabled and Bound to 0.0.0.0:8088

HTTPS Server Enabled and Bound to 0.0.0.0:8089

Enabled URI's:
/httpstatus => Asterisk HTTP General Status
/phoneprov/... => Asterisk HTTP Phone Provisioning Tool
/static/... => Asterisk HTTP Static Delivery
/ari/... => Asterisk RESTful API
/ws => Asterisk HTTP WebSocket

Enabled Redirects:
  None.
```

```
asterisk -rvvv -x 'sip show peers'
Name/username             Host                                    Dyn Forcerport Comedia    ACL Port     Status      Description                      
1060/1060                 (Unspecified)                            D  Auto (No)  No             0        Unmonitored                                  
1061/1061                 (Unspecified)                            D  Auto (No)  No             0        Unmonitored                                  
2 sip peers [Monitored: 0 online, 0 offline Unmonitored: 0 online, 2 offline]
```
```

**Comment by antirek** [comment]
good idea, try describe all steps of your way.

**Comment by mckaygerhard** [comment]
ok let's do taht: in the wiki i'll post a guitde to setup asterisk, sip.js and your example in a new instalation, alter you will point me where ai have the errors!  right? that's good?

**Comment by antirek** [comment]
Let's try. Good docs for server up https://sipjs.com/guides/server-configuration/

**Comment by mckaygerhard** [comment]
i already done all of those steps with asterisk, asterisk works perfecty and also i test in two ways: compiling exactly as the docs said, and also using my own build package (certified works) backported from debian... i upload tomorrow  git commits that you can see!

that docs are not so "good" at all, last lines are very general...

**Comment by mckaygerhard** [comment]
now i finished, you said @antirek "too many steps, i made more easy" but puff this demo are all neither easy to use.. i cannot make it work.. also i must adivinate how to setup, was usin that "on-line" tedious tools.. npm etc etc too many network dependences.. 

now we have a decent starter point:
  - [x] INSTALL.md : file with minimal setup to use this example repository
   - [x] started steps, how to clone and get files,  767a7fe212c43c9f805a68a6535afabed81e70fd
   - [x] configure as need 31b79af3159504612e11aad4d20f65bf387a97ce
   - [x] deploy steps, where top put and see the results 21dfa5f2f627ff9515267846c76f33bf043e6922

i need from you
   - [ ] alternate setup to using the SIP.js ws server and use own deploy withou complicated bower/npm
   - [ ] satrted steps to configure to use with SIPjs ??? a more cheao and thin example!
   - [ ] release type without use git, npm and bower, PLEEEAAASSEEE! just wget and that's all!!!

**Comment by antirek** [comment]
push your install instructions direct to owo-phone repo, ok?

**Comment by mckaygerhard** [comment]
ok but now that my clone project works, the conection does not happned! do you see the error give me now?

in the console js shows: hat Firefox cannot establish a connection in server at wss://10.101.10.197:8089/ws. sip.js:827

maybe i just make only listen in exact ip interfaces? can change to bind to "0.0.0.0" in configuration files?

**Comment by mckaygerhard** [comment]
closed by mergin of #4

**Comment by mckaygerhard** [comment]
i mean, close with #4 due issue was about documentation, but some errors are still hapened..
